build.xml

```diff
@@ -34,18 +34,20 @@
 	<!-- Building Nutch                                         -->
 	<!-- ====================================================== -->
 	<target name="compile" 
-		depends="nutch-compile, compile-typo3-plugins" 
-		description="--> build Nutch" 
+		depends="nutch-compile, compile-typo3-plugins"
+		description="--> build Nutch"
 	/>
 
-	<target name="nutch-compile" depends="init, compile-checkout-src, compile-patch-nutch">
+	<target name="nutch-compile" depends="init, compile-checkout-src">
```

Comment:
What was our position regarding the patch that allows us to exclude parts of the websites?

The original ticket is still open and there seems to be no alternative solution:
https://issues.apache.org/jira/browse/NUTCH-585

	
Disabling the patch is ok from my point of view, but I suggest that we create a ticket as a reminder.

Reply:
Will merge this for now. [Ticket](https://redmine.dkd.de/issues/162035) to reintegrate this patch has been created